SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -680,6 +682,15 @@ bool CMultizoneDriver::Monitor(unsigned long TimeIter){
 
     /*--- Check whether the outer time integration has reached the final time ---*/
     TimeConvergence = GetTimeConvergence();
+
+    if(!cauchyTimeConverged && TimeConvergence && config_container[iZone]->GetTime_Marching() == DT_STEPPING_2ND){
```

Comment:
When you merge develop this becomes:
`config_container[iZone]->GetTime_Marching() == TIME_MARCHING::DT_STEPPING_2ND`
because of #1242

Reply:
Hi Tobi, 
did the change, thanks for the input. 

Cheers